dev/packages/beats/aws-0.0.1/dataset/transitgateway/fields/package-fields.yml

```diff
@@ -0,0 +1,30 @@
+- name: cloud.account.name
+  type: keyword
+  description: >
+    The cloud account name or alias used to identify different entities in a multi-tenant environment.
+- name: aws
+  type: group
+  description: >
+  fields:
+    - name: tags.*
+      type: object
+      object_type: keyword
+      object_type_mapping_type: "*"
+      description: >
+        Tag key value pairs from aws resources.
+    - name: s3.bucket.name
+      type: keyword
+      description: >
+        Name of a S3 bucket.
+    - name: dimensions.*
+      type: object
+      object_type: keyword
+      object_type_mapping_type: "*"
+      description: >
+        Metric dimensions.
+    - name: '*.metrics.*.*'
```

Comment:
@skh @neptunian Just stumbled over this one here. I wonder if we will be able to handle this correctly in EPM?

Reply:
@ruflin should this package exist in the example directory?  they two packages are different.  otherwise I can manually test it.  I know that there are field names like `system.network_summary.ip.*` are showing up as index pattern fields so if it begins with a * it should be fine.  I can add it to the unit tests to be sure.

Reply:
The package does not show up, because this one is older then the example package. But we could copy these fields over to the example one to have it more complete.